charts/tfy-inferentia-operator/values.yaml

```diff
@@ -39,3 +39,35 @@ devicePlugin:
                   - trn1.2xlarge
                   - trn1.32xlarge
                   - trn1n.32xlarge
+
+## @section Configuration for the scheduler responsible for scheduling neuron pods
+## @param scheduler.enabled Enable Scheduler.
+## @param scheduler.schedulerName Name of the scheduler.
+## @param scheduler.image K8s Scheduler image.
+## @param scheduler.resources.requests.cpu CPU request for K8s scheduler.
+## @param scheduler.resources.requests.memory Memory request for K8s scheduler.
+## @param scheduler.extension.image Neuron scheduler extension image.
+## @param scheduler.extension.resources.requests.cpu CPU request for Neuron scheduler extension.
+## @param scheduler.extension.resources.requests.memory Memory request for Neuron scheduler extension.
+scheduler:
+  enabled: true
+  schedulerName: neuron-scheduler
```

Comment:
nit
```suggestion
  name: neuron-scheduler
```

Reply:
I want to keep this specific. `name` is too generic.